### PR TITLE
feat: add support for windows node metrics

### PR DESCRIPTION
**Pull request by sumo-drosiek**
### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -1123,7 +1124,8 @@ kube-prometheus-stack:
     ## Put here the new name if you want to override the full name used for Prometheus Node exporter components.
     # fullnameOverride: ""
 
-    nodeSelector: {}
+    nodeSelector:
+      kubernetes.io/os: linux
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:1128]: This configuration for prometheus node exporter and we should explicitly leave it as here

File: docs/README.md [hunk 2]
```diff
@@ -123,3 +123,16 @@ environment, don't hesitate to open an [issue][issues] describing them.
 ### Falco support
 
 Falco is embedded in this Helm Chart for user convenience only - Sumo Logic does not provide production support for it.
+
+### Windows nodes support
+
+Windows nodes are supported only for metrics collection. To enable it, add the dollowing configuration to your `user-values.yaml`
```
Comment (sumo-drosiek) [inline docs/README.md:129]:
```suggestion
Windows nodes are supported only for metrics collection. To enable it, add the following configuration to your `user-values.yaml`
```

File: docs/windows.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Windows support
+
+Windows support is under evaluation. This document describes issues which should be resolved before claiming full support for windows.
```
Comment (aboguszewski-sumo) [inline docs/windows.md:3]: What do you mean by `under evaluation`?

Comment (sumo-drosiek) [reply docs/windows.md:3]: That we are validating how it works, and we are going to make it working, but we are not going to do any claims in nearest future :sweat_smile:

File: docs/README.md [hunk 4]
```diff
@@ -123,3 +123,16 @@ environment, don't hesitate to open an [issue][issues] describing them.
 ### Falco support
 
 Falco is embedded in this Helm Chart for user convenience only - Sumo Logic does not provide production support for it.
+
+### Windows nodes support
+
+Windows nodes are supported only for metrics collection. To enable it, add the following configuration to your `user-values.yaml`
```
Comment (swiatekm-sumo) [inline docs/README.md:129]: Let's add a note that this is experimental. This note should not go away until we have E2E tests checking this.

File: docs/windows.md (hunk 3 above)
Comment (sumo-drosiek) [reply docs/windows.md]: I add that this is experimental, but left also that it is under evalutation. I think it's important information that it is not clear yet how much we do support windows or can support it with minimal amount of work

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**